### PR TITLE
Run3-gex74X Try to resolve the overlap in PPS

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Try to resolve the overlap in PPS.  It changes positions of some of the volumes.

#### PR validation:

Tested with overlap check tools

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/VeryForwardData/data/CTPPS_Timing_Horizontal_Pot/2021/v1/CTPPS_Timing_Horizontal_Pot.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <UnionSolid name="Primary_Vacuum">
+      <rSolid name="Primary_Vacuum_nocut"/>
+      <rSolid name="Window_Cut"/>
+      <Translation x="0*mm" y="0*mm" z="-([plane_length]+[cut_depth])/2"/>
+    </UnionSolid>
+    <SubtractionSolid name="Bottom_Wall">
+      <rSolid name="Bottom_Wall_nocut"/>
+      <rSolid name="Window_Cut"/>
+      <Translation x="0*mm" y="0*mm" z="[bottom_wall_thickness]/2-[cut_depth]/2"/>
+  </SubtractionSolid>
```
Comment (cvuosalo) [inline Geometry/VeryForwardData/data/CTPPS_Timing_Horizontal_Pot/2021/v1/CTPPS_Timing_Horizontal_Pot.xml:28]: Indentation is wrong here.

File: Geometry/VeryForwardData/data/CTPPS_Timing_Horizontal_Pot/2021/v1/CTPPS_Timing_Horizontal_Pot.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cvuosalo) [inline Geometry/VeryForwardData/data/CTPPS_Timing_Horizontal_Pot/2021/v1/CTPPS_Timing_Horizontal_Pot.xml:1]: Did you want to change this file? I did a diff and found no significant changes.

Comment (bsunanda) [reply Geometry/VeryForwardData/data/CTPPS_Timing_Horizontal_Pot/2021/v1/CTPPS_Timing_Horizontal_Pot.xml:1]: I was working with 2 xml files and changed them several times. At the end I was happy by changing only 1 file. Let me delete the unchanged file and redo the cfi's

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by srimanob** [comment]
Could you please clarify the naming convention you are using, i.e 2018 in "CTPPS_Diamond_2018" with "2021". I understand that 2021 is used to represent Run-3. Why don't we start to make CTPPS_Diamond_2021?

`'Geometry/VeryForwardData/data/CTPPS_Diamond_2018/CTPPS_Diamond_Detector_Assembly/2021/v1/CTPPS_Diamond_Detector_Assembly.xml'`

**Comment by forthommel** [comment]
This is probably where the PPS RECO people can be ping'ed!
@fabferro @jan-kaspar

**Comment by fabferro** [comment]
@civanch we are considering a reordering in this directory while implementing the real new geometry for run3 detectors.
This should happen soon, probably not in a single step. 
Unfortunately there are several files to be considered and the way they have been added have not followed good criteria.

**Comment by civanch** [comment]
@fabferro , we need to make a choice: we agree with the fix of @bsunanda , or we ask to move new files to another directory. This needs to be decided just today. Can you, please, comment?

**Comment by cvuosalo** [comment]
I think this PR should be merged as is. The re-organization of file names is a separate issue. It's important to get the overlaps fixed now.

**Comment by fabferro** [comment]
> 
> 
> I think this PR should be merged as is. The re-organization of file names is a separate issue. It's important to get the overlaps fixed now.

Yes. I agree. Please go on with this fix.

**Comment by civanch** [comment]
@srimanob , would you agree that directory structure will be sort out in the following PRs?

**Comment by srimanob** [comment]
+Upgrade

This PR is to fix overlap in PPS. As agreed in the PR discussion, this PR fix is merged first. Then we expect the re-organization of filenames, directories in the new Run-3 geometry.

**Comment by qliphy** [comment]
+1

**Pull request merged**